docs/en/sql-reference/statements/insert-into.md

```diff
@@ -148,7 +149,18 @@ Result:
 └────┴──────┘
 ```
 
-### Inserting into Table Function
+### Multiple files with FROM INFILE using globs
+
+This example is very similar to the previous one but inserts from multiple files using `FROM INFILE 'input_*.csv`.
+
+```bash
+echo 1,A > input_1.csv ; echo 2,B > input_2.csv
+clickhouse-client --query="CREATE TABLE infile_globs (id UInt32, text String) ENGINE=MergeTree() ORDER BY id;"
+clickhouse-client --query="INSERT INTO infile_globs FROM INFILE 'input_*.csv' FORMAT CSV;"
```

Comment:
I think it makes sense to show that not only * is supported. I'm not sure that I wrote them correctly, but to show what I suggest
```suggestion
clickhouse-client --query="INSERT INTO infile_globs FROM INFILE 'input_*.csv' FORMAT CSV;"
clickhouse-client --query="INSERT INTO infile_globs FROM INFILE 'input_{1,2}.csv' FORMAT CSV;"
clickhouse-client --query="INSERT INTO infile_globs FROM INFILE 'input_??.csv' FORMAT CSV;"

```

Reply:
good suggestion, I moved it into a tip.  Thanks!